### DISCUSSION TITLE
安装完成idea后，怎么过桥呢

### DISCUSSION THREAD

**Discussion by breakToT** [ByteLegend/ByteLegend#52] [General]
不知道怎么继续了。刚刚安装完成IDEA，不知道往哪里走，过桥怎么点呢

**Comment by breakToT** [comment]
emmmm，搞不懂了。

**Reply by blindpirate** [reply]
桥那边的代码还没开发完，守桥的士兵已经告诉你了。

**Reply by breakToT** [reply]
所以我拿到安装IDE的星星后应该干嘛呢。

**Reply by gtn1024** [reply]
> 所以我拿到安装IDE的星星后应该干嘛呢。

Waiting for development

**Reply by breakToT** [reply]
> > 所以我拿到安装IDE的星星后应该干嘛呢。
> 
> Waiting for development

期待

**Reply by ashuanger** [reply]
点击安装Java为什么没反应呢

**Reply by blindpirate** [reply]
@ashuanger 你需要走到咖啡机边上才能使用咖啡机。

**Reply by ashuanger** [reply]
> @ashuanger 你需要走到咖啡机边上才能使用咖啡机。

谢谢，已经完成了，原来需要走到门口才能进入酒吧
